script/build.js

```diff
@@ -125,11 +125,9 @@ smith.metadata({ buildtype: options.buildtype });
 const ignore = require('metalsmith-ignore');
```

Comment:
While you're editing this, there are some comments that could be deleted:
  * the `TODO(awong):` above can go away since that issue is closed
  * the "To use" can mostly go away since that code is already below. We can just add
```
To block an app from production, add the following to the below list:
  ignoreList.push('<path-to-content-file>');
```